### ISSUE TITLE
Option `ignore_http_errors`

### ISSUE THREAD

**Issue by ricardopereira** [rc0x03/node-osmosis#57]
I set `ignore_http_errors` option to `true` but it doesn't work.
Any clue?

```
var osmosis = require('osmosis');

var options = {
  ignore_http_errors: true
};

osmosis
.get('http://www.jumbo.pt/Frontoffice/ContentPages/BrochureDetail.aspx?C=FL_8252000&loop=1', options)
.find('div.produtoGrelha')
.set({
	'title': 'div.titPromo',
	'price': 'div.preco+span.centimos',
	'link': 'div.titPromo @href'
})
.data(function(listing) {
	console.log(listing)
})
.error(console.log)
```

**Comment by rc0x03** [comment]
The options argument should be the last argument for `get`. You must specify that the query params are null, otherwise your options will be sent as query params. so it should be `get('http://....', null, options)`

In the future I'd like to deprecate configuration via arguments and require the use of `.config` and `.headers` instead. So alternatively you can call `osmosis.config(...)` to set a global configuration or call `.config(...)` immediately after `.get`, which will set options for `.get` and everything after that.

**Comment by ricardopereira** [comment]
Oh, maybe it is better to adopt the new way.

So, I do:
```
var osmosis = require('osmosis');

osmosis
.get('http://www.jumbo.pt/Frontoffice/ContentPages/BrochureDetail.aspx?C=FL_8252000&loop=1')
.config({
  ignore_http_errors: true
})
.find('div.produtoGrelha')
.set({
    'title': 'div.titPromo',
    'price': 'div.preco+span.centimos',
    'link': 'div.titPromo @href'
})
.data(function(listing) {
    console.log(listing)
})
.error(console.log)
```

but I still get:

```
.../node_modules/osmosis/index.js:188
            if (res.statusCode >= 400 && res.statusCode < 500 && opts.ignore_http_errors !== true)
                   ^

TypeError: Cannot read property 'statusCode' of undefined
```

**Comment by rc0x03** [comment]
This is caused by improper handling of some HTTP errors. The error in this case is "Max redirects reached. Possible loop in: [URL]".

This will be fixed in the next version.

**Comment by rc0x03** [comment]
Fixed in 0.1.1

**Comment by ricardopereira** [comment]
:+1:
